app/components/MixerVolmeter.vue.ts

```diff
@@ -110,6 +128,38 @@ export default class MixerVolmeter extends TsxComponent<MixerVolmeterProps> {
 
     this.setCanvasWidth();
     this.canvasWidthInterval = window.setInterval(() => this.setCanvasWidth(), 500);
+    requestAnimationFrame(() => this.onRequestAnimationFrameHandler());
+  }
+
+  /**
+   * Render volmeters with FPS capping
+   */
+  private onRequestAnimationFrameHandler() {
+    // init first rendering frame
+    if (!this.frameNumber) {
+      this.frameNumber = -1;
+      this.firstFrameTime = performance.now();
```

Comment:
The callback to `requestAnimationFrame` is passed a `DOMHighResTimeStamp` in its arguments that is meant to be used for calculations like this.  Seems more logical and possibly higher performance to use that rather than making several `performance.now()` calls inside this function.